src/Perl6/Actions.nqp

```diff
@@ -7589,8 +7589,19 @@ class Perl6::Actions is HLL::Actions does STDActions {
             $dynbind.nosink(1);
             make $dynbind;
         }
-        # XXX Several more cases to do...
-        else {
+        # Everything else is a (re)binding error.  Check the types to give a more specific msg
+        elsif nqp::istype($target, QAST::WVal)
+        && $target.value.raku eq $target.node {    # A type (class, role, etc)
+            $*W.throw($/, ['X', 'Bind', 'Rebind'], :target($target.value.raku), :is-type(1))
+        }
+        elsif nqp::istype($target, QAST::WVal) {   # A constant
+            $*W.throw($/, ['X', 'Bind', 'Rebind'], :target(nqp::escape($target.node)))
+        }
+        elsif nqp::istype($target, QAST::Op)
+        && $target.op eq 'ifnull' {                # Code (subs, regex, etc)
```

Comment:
Nitpicking on indentations here and above, at line 7594.

Reply:
> Nitpicking on indentations here and above, at line 7594.

Do you mean the fact that the `&&` isn't indented relative to the line above?  That's not my preferred style either, but it seems to be the most common one in use in this file, including in this fn at line 7581.  I'm happy to change it if you'd like, but I figured that opting for local consistency was the lesser evil here.

Reply:
I'd better change the other indentations. Looks like either 7581 and others are artifacts of incorrectly expanded tabs, or a weird habit of Zoffix. Anyway, it seems we have such a diversity of styles in the file that it's hard to stick to a single one. So, let's choose something readable. :) My own preference is to have logic op in front of its RHS, indented. It's also what Liz does. Looks like Larry and Jonathan prefer the op to follow its RHS. So, just choose whichever one is better for you. :)